tests/handlers/test_base_handler.py

```diff
@@ -1256,6 +1257,13 @@ def test_max_bytes_impossible(self):
         expect(response.code).to_equal(200)
         expect(len(response.body)).to_be_greater_than(1000)
 
+    def test_meta_with_exif_orientation(self):
+        response = self.fetch('/unsafe/meta/0x0/Giunchedi%2C_Filippo_January_2015_01-cmyk-orientation-exif.jpg')
+        expect(response.code).to_equal(200)
+        obj = loads(response.body)
+        expect(obj['thumbor']['target']['width']).to_equal(533)
+        expect(obj['thumbor']['target']['height']).to_equal(800)
```

Comment:
If I get it right, then Giunchedi%2C_Filippo_January_2015_01-cmyk-orientation-exif.jpg is a landscape photography which is exif rotated to portrait (allthough the motiv is landscape?). Isn't this very confusing for an example image in testdata?

So the phyical/real image dimensions are 800/533. Its Orientation is right top (6), so it will be rotated 90 degrees to the right.. That makes width 533/800.  So this test makes sense to me - not the image, though :)